pom.xml

```diff
@@ -93,6 +105,7 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
+            <version>42.2.16</version>
```

Comment:
Somehow there was no version spec'ed for this - I didn't realize Maven would let it compile in that case.